docs/troubleshooting.adoc

```diff
@@ -0,0 +1,19 @@
+== AeroGear Android Troubleshooting
+
+=== invokedynamic requires --min-sdk-version >= 26
```

Comment:
I will change title to be less cryptic. `Problems with building on Android SDK lower than 26` ?

Reply:
Not sure because that is **exactly** what logcat says. See: https://github.com/aerogear/aerogear-android-sdk/issues/55

Reply:
Perhaps "Build error : invokedynamic requires --min-sdk-version >= 26" will be a better title?